synapsis/src/test/java/com/soen/synapsis/unit/appuser/AppUserServiceTest.java

```diff
@@ -77,28 +77,26 @@ void signUpUserWithExistingEmailThrows() {
 
     @Test
     void markCandidateToRecruiterSucceeds() {
-        AppUser appUserCandidate = new AppUser(1L, "Joe Man", "1234", "joecandidate@mail.com", Role.CANDIDATE);
-        AppUser appUserCompany = new AppUser(2L, "Joe Man", "1234", "joecompany@mail.com", Role.COMPANY);
-        AppUserDetails loggedInAppUser = new AppUserDetails(appUserCompany);
-        appUserRepository.save(appUserCandidate);
-        appUserRepository.save(appUserCompany);
+        AppUser candidateUser = new AppUser(1L, "Joe Man", "1234", "joecandidate@mail.com", Role.CANDIDATE);
```

Comment:
These tests where we call appUserRepository.save should be integration tests (just move them over to the appropriate folder).

Reply:
I took those calls out. It should be unit tests.